ansible/roles/lib_aos_modules/library/aos_rhsm_repos.py

```diff
@@ -151,7 +154,7 @@ def needs_update(self):
 
     def check_identity(self):
         """verify that this system is entitled"""
-        rcode, _, _ = self.module.run_command([RHSM_BIN, 'identity'])
+        rcode, _, _ = self.module.run_command(RHSM_ENTRYPOINT + ['identity'])
```

Comment:
Still coming up to speed on python, but yes. RHSM_BIN was just a string; the new RHSM_ENTRYPOINT is a list. So:  
list(RHSM_ENTRYPOINT) + list(['identity']) = list(a combination of both) 

It is this combination that is passed to run_command.